rpc/rest/security/model/http-security/httpsecurity.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+// Package http_security provides basic definition of user and permission group
+package http_security;
+
+message User {
+    string name = 1;
+    string password = 2;
```

Comment:
It should be explicit that the password is hashed, please name it `hashed_password`.